cql3/untyped_result_set.hh

```diff
@@ -147,23 +172,27 @@ public:
         get_set_data<V>(name, std::inserter(res, res.end()), valtype);
         return res;
     }
-    const std::vector<lw_shared_ptr<column_specification>>& get_columns() const {
-        return _columns;
+    const cql3::metadata& get_metadata() const {
+        return _metadata;
     }
+    const std::vector<lw_shared_ptr<column_specification>>& get_columns() const;
 };
 
 class result_set;
 
 class untyped_result_set {
 public:
     using row = untyped_result_set_row;
-    typedef std::vector<row> rows_type;
+    using rows_type = std::vector<row>;
     using const_iterator = rows_type::const_iterator;
     using iterator = rows_type::const_iterator;
 
     untyped_result_set(::shared_ptr<cql_transport::messages::result_message>);
-    untyped_result_set(const cql3::result_set&);
+    untyped_result_set(const schema&, foreign_ptr<lw_shared_ptr<query::result>>, const cql3::selection::selection&, const query::partition_slice&);
+    untyped_result_set(cql3::result_set&&);
+    untyped_result_set(cql3::result&&);
```

Comment:
Where are those two constructors implemented? I couldn't find their definitions. Are they used at all?

Reply:
They are not supposed to be there. Sorry, forgot to prune...